demos/00-helpers/src/helpers.js

```diff
@@ -1,3 +1,26 @@
+import { setCORS } from "google-translate-api-browser";
+const translate = setCORS("http://cors-anywhere.herokuapp.com/");
+
+function schedule_a_mock_reply(message_text, callback)
+{
+	message_text = g_scm2host(message_text);
+	console.log("translating " + message_text.toString() + "...");
```

Comment:
If `message_text` is a scheme string, not need to call `toString` after `g_scm2host`